src/test/java/org/fcrepo/sword/http/SWORDServiceResourceTest.java

```diff
@@ -13,24 +13,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.fcrepo.sword.integration;
+package org.fcrepo.sword.http;
 
+import org.apache.abdera.model.Service;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import static org.junit.Assert.assertEquals;
 
 /**
  * @author claussni
- * @date 19.02.15.
+ * @date 18.02.15.
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration("/spring-test/test-container.xml")
-public class AbstractSWORDProviderIT {
+@ContextConfiguration("/spring/sword.xml")
+@WebAppConfiguration
+public class SWORDServiceResourceTest {
```

Comment:
That's not intended. This should clearly be a unit test. I tried to use the Spring DI container but the problem here is that there is no scope `request`. Setting the scope of `SWORDServiceResource` to `prototype` resolves this.